src/tools/kotlin-entity-generator.ts

```diff
@@ -239,15 +131,43 @@ ${expression}
     const fieldCount = Object.keys(this.node.schema.fields).length;
     const blocks: string[] = [];
 
+    const fieldVals: string[] = [];
+    for (const {field, type, isCollection, escaped, defaultVal, nullableType} of this.schemaDescriptor.fields) {
+      if (this.opts.wasm) {
+        // TODO: Add support for collections in wasm.
+        assert(!isCollection, 'Collection fields not supported in Kotlin wasm yet.');
+        fieldVals.push(
+          `var ${escaped} = ${escaped}\n` +
```

Comment:
Instead of concatenating strings, let's make these into a single block. WDYT?

Reply:
We can then remove the `\n`s at the ends of the strings.

Reply:
Done!